nbgrader/docs/source/user_guide/philosophy.rst

```diff
@@ -85,16 +85,15 @@ work).
 Configuration files
 ~~~~~~~~~~~~~~~~~~~
 
-You will almost always need a configuration file as you are using nbgrader. At a minimum, this configuration file will contain information such as the names of the assignments for your class and the names of your students (though you may also add these by hand using the command line, see :doc:`creating_and_grading_assignments`). A basic config file would live at ``{course_directory}/nbgrader_config.py`` and might look like:
+You may need a configuration file as you are using nbgrader to customize
+nbgrader's behavior. If you are using nbgrader with :doc:`JupyterHub
+</configuration/jupyterhub_config>`, you will at minimum need to set the course
```

Comment:
...you ~~will at minimum need~~ must set the course id in the `nbgrader_config.py` configuration file.